public/pages/VisualCreatePolicy/components/FlyoutFooter/FlyoutFooter.tsx

```diff
@@ -24,7 +25,7 @@ const FlyoutFooter = ({ edit, action, disabledAction = false, onClickCancel, onC
     </EuiFlexItem>
     <EuiFlexItem grow={false}>
       <EuiButton disabled={disabledAction} onClick={onClickAction} fill data-test-subj="flyout-footer-action-button">
-        {!save ? `${edit ? "Edit" : "Add"} ${action}` : save ? "Save" : "Create"}
+        {restore ? "Restore snapshot" : !save ? `${edit ? "Edit" : "Add"} ${action}` : save ? "Save" : "Create"}
       </EuiButton>
```

Comment:
This is a bit tricky at first glance.  I did this to include "Restore snapshot" so it would appear on the button in the Restore Flyout